utils/cosmos/iscn/sign.ts

```diff
@@ -53,12 +54,22 @@ export function formatISCNTxPayload(payload: ISCNRegisterPayload): ISCNSignPaylo
           value: a.address,
         }))
 
+      const wallet = identifiers.length ? identifiers[0].value : likerIdsAddresses[i]
+
+      const likerIdentifiers:any = likerIds.map((a: any) => ({
```

Comment:
Should this be `likerIds[i]` or `likerIds.map()`? Not sure the original meaning for `likerIds` but seems url is `likerIds[i]`